win32/ctags_vs2013.vcxproj

```diff
@@ -338,8 +344,12 @@
     <ClInclude Include="..\parsers\make.h" />
     <ClInclude Include="..\parsers\perl.h" />
     <ClInclude Include="..\parsers\tcl.h" />
+    <ClInclude Include="resource.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ResourceCompile Include="ctags.rc" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Visual Studio removes the last CRLF from `*.vcxproj` and `*.vcxproj.filters`.
@masatake I think `src/src-check` needs to be adjusted for that.

Reply:
Done.

Reply:
Thank you.